docker/local/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update
 RUN apt-get install -y sudo curl vim jq
 
 # Install dependencies
-COPY docker/lite/install_dependencies.sh /vt/dist/install_dependencies.sh
+COPY docker/utils/install_dependencies.sh /vt/dist/install_dependencies.sh
```

Comment:
Do we still need to have this script? It installs a number of packages, but are all those packages still relevant? or should we clean more in there too?

Reply:
This script is now only used by the `local` and `vttestserver` images so it is fine to keep it as it is since we still require mysqld/percona-server on those images.